client/src/components/Dashboard.js

```diff
@@ -141,10 +143,31 @@ class Dashboard extends Component {
     });
   };
 
+  handleToggle = languages => {
+    this.setState({ toggled: !this.state.toggled });
+  };
+
+  getFilteredDocs = (documents, languages) => {
+    if (!this.state.toggled) {
+      return documents;
```

Comment:
You don't need to pass the documents as a parameter to this function, you can use directly `this.state.documents` inside here :)

Reply:
done